tests/Tests/600__ai_apps/640__rhoam/640__rhoam_installation.robot

```diff
@@ -0,0 +1,84 @@
+*** Settings ***
+Resource        ../../../Resources/Page/LoginPage.robot
+Resource        ../../../Resources/Common.robot
+Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../../Resources/Page/ODH/AiApps/Rhoam.resource
+Resource        ../../../../tasks/Resources/RHODS_OLM/uninstall/uninstall.robot
+Resource        ../../../../tasks/Resources/RHODS_OLM/uninstall/oc_uninstall.robot
+Resource        ../../../../tasks/Resources/RHODS_OLM/config/cluster.robot
+Library         ../../../../libs/Helpers.py
+Library         SeleniumLibrary
+Library         OpenShiftLibrary
+Suite Setup     RHOAM Suite Setup
+Suite Teardown  RHOAM Suite Teardown
+
+
+*** Test Cases ***
+Verify RHOAM Can Be Installed
+    [Documentation]    Verifies RHOAM Addon can be successfully installed
+    [Tags]  Tier3
+    ...     ODS-273
+    ${cluster_id}=   Get Cluster ID
+    ${cluster_name}=   Get Cluster Name     cluster_identifier=${cluster_id}
+    Install Rhoam Addon    cluster_name=${cluster_name}
```

Comment:
I think rhoam addon could be installed in the `Suite Setup`  (to be able to run ODS-1136 independently of  ODS-273) same as it is done in tests/Tests/500__jupyterhub/minimal-cuda-test.robot

Reply:
Following the approach in minimal-cuda-test.robot the time will be the same (installation is in Suite Setup and ODS-1136 only executes Pass)

Reply:
hm basically ODS-273 would become a Suite setup. So it wouldn't change anything here. Because to run ODS-1136 we will run a setup which consists of ODS-273, so ODS-1136 will no be really independent. The change would bring only a cosmetic difference which IMO it's not worth now